<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basketDialog</class>
 <widget class="QDialog" name="basketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>배달 애플리케이션 - WAGU</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>14</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>장바구니</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="basketList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>386</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>결제 하실 내역</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>굴림</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="useCouponBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>쿠폰 사용</string>
   </property>
  </widget>
  <widget class="QComboBox" name="couponBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="payBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>결제 후 주문하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>결제 방법</string>
   </property>
  </widget>
  <widget class="QComboBox" name="methodBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>560</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>일회용 식기 수령 여부</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tableWareBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>94</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
